server/db/index.js

```diff
@@ -4,115 +4,134 @@ const db = require('./db')
+      name: 'Daisy Sour Cream, 16 oz',
+      price: 2.39, category: 'Dairy, Cheese & Eggs',
+      quantity: 10,
+      descripton: 'What makes Daisy America’s favorite brand of sour cream? It could be the fresh and delicious taste, the rich and creamy texture, or that it’s made from simple, wholesome ingredients - with no additives or preservatives.',
       ImgUrl: '/images/sze_sourcream.jpg'
     }),
     Product.create({
-      name: 'Happy Belly Mozzarella String Cheese, 12 Count, 12 Ounce Pack', 
-      price: 2.89, category: 'Dairy, Cheese & Eggs', 
-      quantity: 10, 
-      descripton: 'Happy Belly String Cheese gives you 12 individually wrapped sticks of low moisture, part-skim milk mozzarella cheese. They contain milk and are a good source of calcium. Add some to a lunch box or keep them handy for fun, satisfying snacks.', 
+      name: 'Happy Belly Mozzarella String Cheese, 12 Count, 12 Ounce Pack',
+      price: 2.89, category: 'Dairy, Cheese & Eggs',
+      quantity: 10,
+      descripton: 'Happy Belly String Cheese gives you 12 individually wrapped sticks of low moisture, part-skim milk mozzarella cheese. They contain milk and are a good source of calcium. Add some to a lunch box or keep them handy for fun, satisfying snacks.',
       ImgUrl: '/images/sze_stringcheese.jpg'
     }),
     Product.create({
-      name: 'Califia Farms - Almond Milk, Unsweetened, 48oz,', 
-      price: 4.39, category: 'Dairy, Cheese & Eggs', 
-      quantity: 10, 
-      descripton: 'Our artisanally crafted Almondmilk that\'s 100% sugar-free and has creamy, pure taste will make it a quick staple in your fridge, coffee, smoothies, recipes and more.', 
+      name: 'Califia Farms - Almond Milk, Unsweetened, 48oz,',
+      price: 4.39, category: 'Dairy, Cheese & Eggs',
+      quantity: 10,
+      descripton: 'Our artisanally crafted Almondmilk that\'s 100% sugar-free and has creamy, pure taste will make it a quick staple in your fridge, coffee, smoothies, recipes and more.',
       ImgUrl: '/images/sze_almondmilk.jpg'
     }),
     Product.create({
-      name: 'Lactaid 2% Reduced Fat Milk, 96 fl oz', 
-      price: 6.59, category: 'Dairy, Cheese & Eggs', 
-      quantity: 10, 
-      descripton: 'Get essential nutrients like calcium, protein, and Vitamin D from the milk your taste buds and your tummy can agree on.', 
+      name: 'Lactaid 2% Reduced Fat Milk, 96 fl oz',
+      price: 6.59, category: 'Dairy, Cheese & Eggs',
+      quantity: 10,
+      descripton: 'Get essential nutrients like calcium, protein, and Vitamin D from the milk your taste buds and your tummy can agree on.',
       ImgUrl: '/images/sze_lactaidmilk.jpg'
     })
   ])
+
+  //HARD CODED AN ORDER TIED TO OUR USER NAMED CODY
+  Order.create({
+    userId: 2,
+    isOpen: true,
+    products: [
+      {
```

Comment:
Hard coded a user just to get a barebones order table for a user . 

I hard coded this order table to belong to CODY